### PR TITLE
Add back https settings and rules

### PR DESCRIPTION
**Pull request by markry-kainos**
Access via https://moneyclaims.aat.platform.hmcts.net/ still not working. DNS has been updated. Thinking about this... we surely do need the https settings. The app service is set to https only so this will redirect back to app gw and not work. Or am I missing something here?

### CODE REVIEW

**Review by timja — CHANGES_REQUESTED** [review]
Right forgot about that, you'll need another option for that then
https://github.com/hmcts/cnp-module-waf#using-authentication-certificates
`use_authentication_cert = true`

Also would be ideal to separate out these changes as they are unrelated

File: appgw.tf [hunk 1]
```diff
@@ -168,6 +191,18 @@ module "appGwSouth" {
...
+      name                                = "citizen-https-probe"
+      protocol                            = "Https"
+      path                                = "${var.health_check}"
+      interval                            = 30
+      timeout                             = 30
+      unhealthyThreshold                  = 5
+      pickHostNameFromBackendHttpSettings = "false"
+      backendHttpSettings                 = "citizen-backend-443"
+      host                                = "${local.paybubble_backend_hostname}"
+      healthyStatusCodes                  = "200-404"                             // MS returns 404 on /, allowing more codes in case they change it
```
Comment (timja) [inline appgw.tf]: you only want 200-299, or just 200, the 404 is for bulk scanning

File: appgw.tf [hunk 2]
```diff
@@ -168,6 +189,18 @@ module "appGwSouth" {
...
+    {
+      name                                = "citizen-https-probe"
+      protocol                            = "Https"
+      path                                = "${var.health_check}"
+      interval                            = 30
+      timeout                             = 30
+      unhealthyThreshold                  = 5
+      pickHostNameFromBackendHttpSettings = "false"
+      backendHttpSettings                 = "citizen-backend-443"
+      host                                = "${local.paybubble_backend_hostname}"
```
Comment (markry-kainos) [inline appgw.tf]: oops....

**Review by timja — APPROVED** [review]
good once you sort out your bubbling

**Comment by timja** [comment]
A good idea might be to add something like:
https://github.com/hmcts/cnp-plum-shared-infrastructure/blob/master/Jenkinsfile_CNP#L23
```
onPR {
  sharedInfrastructurePipeline('cmc', environment.nonProdName, 'nonprod', true)
}
```

**Comment by markry-kainos** [comment]
> onPR {
>   sharedInfrastructurePipeline('cmc', environment.nonProdName, 'nonprod', true)
> }

yep - like it, good idea

**Pull request closed without merging**